src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -81,26 +81,15 @@ class Connection
     private $amqpFactory;
     private $autoSetupExchange;
     private $autoSetupDelayExchange;
-
-    /**
-     * @var \AMQPChannel|null
-     */
-    private $amqpChannel;
-
-    /**
-     * @var \AMQPExchange|null
-     */
-    private $amqpExchange;
+    private \AMQPChannel|null $amqpChannel;
+    private \AMQPExchange|null $amqpExchange;
```

Comment:
Please remove all types added to properties: we're going to do that in a dedicated PR.